contracts/masset/platform-integrations/CompoundIntegration.sol

```diff
@@ -239,6 +245,50 @@ contract CompoundIntegration is InitializableAbstractIntegration {
         uint256 exchangeRate = _cToken.exchangeRateStored();
         // e.g. 1e18*1e18 / 205316390724364402565641705 = 50e8
         // e.g. 1e8*1e18 / 205316390724364402565641705 = 0.45 or 0
-        amount = _underlying.mul(1e18).div(exchangeRate);
+       amount = _underlying.mul(1e18).div(exchangeRate);
+    }
+
+    /**
+     * @dev Checks whether a claim should be made
+     * This compares the block.timestamp with a somewhat random time
+     * Adds randomness by muliplying the 1 hour delay between 1x and 3x
+     * @param _bAsset Address for the bAsset
+     */
+    function _checkClaim(address _bAsset)
```

Comment:
We are only selling 1 asset on the Liquidator, so checking the claim for other bassets here will result in nothing being collected. Either we need to decide to sell more than 1 asset on the Liquidator, or specifically only collect for 1 bAsset (dai) here.